app/scripts/modules/core/src/diffs/changes.html

```diff
@@ -3,6 +3,24 @@
   <div class="modal-header has-sticky-headers">
     <h3>Changes to {{$ctrl.nameItem.name}}</h3>
   </div>
+  <div class="modal-body" ng-if="(!$ctrl.previousBuildLink || !$ctrl.currentBuildLink) && $ctrl.buildInfo.ancestor && $ctrl.buildInfo.target">
```

Comment:
make this section fixed and always visible, even if the diff sections are scrollable.